scripts/mysql_system_tables_data.sql

```diff
@@ -26,13 +26,18 @@
 -- a plain character
 SELECT LOWER( REPLACE((SELECT REPLACE(@@hostname,'_','\_')),'%','\%') )INTO @current_hostname;
 
+SET @str = IF (@skip_install_test_db IS NULL, 'CREATE DATABASE IF NOT EXISTS test', 'SET @dummy = 0');
+PREPARE stmt FROM @str;
+EXECUTE stmt;
+DROP PREPARE stmt;
```

Comment:
Why is it in `mysql_system_tables_data.sql` and not in `mysql_system_tables.sql`?

Reply:
because its with the other bits that skip_install_test_db related to test database skipping. no other reason.

Reply:
did you want this moved to ```mysql_system_tables.sql```?